File: test/benchmarks/lua/loop.lua
local start = os.clock()

local N = 10000000
local list = {}

for i = 1, N do
  list[i] = i - 1
end

local sum = 0

for i = 1, N do
  sum = sum + list[i]
end

local seconds = os.clock() - start
print('elapsed: ' .. seconds .. 's')

File: test/benchmarks/node/loop.js
const process = require('process')

const start = process.hrtime.bigint()

const N = 10000000
const list = []

for (let i = 0; i < N; i++) {
  list.push(i)
}

let sum = 0

for (let i = 0; i < N; i++) {
  sum += list[i]
}

const seconds = Number(process.hrtime.bigint() - start) / 1000000000
console.log('elapsed: ' + seconds + 's | ' + sum)
